topostats/unet_masking.py

```diff
@@ -14,6 +14,9 @@
 
 LOGGER = logging.getLogger(LOGGER_NAME)
 
+# pylint: disable=too-many-positional-arguments
+# pylint: disable=too-many-locals
+
 
 # DICE Loss
```

Comment:
Comment is extraneous given the docstring but could the `DICE` acronym (if it is an acronym :thinking: ) be explained in the docstring?

I've never heard of it. The [TensorFlow docs](https://www.tensorflow.org/api_docs/python/tf/keras/losses/Dice) don't explain it in lay terms (although they do give the formula). [This](https://cvinvolution.medium.com/dice-loss-in-medical-image-segmentation-d0e476eb486) gives a nice grahpical overview (although the text doesn't read very clearly).

Reply:
```suggestion
```